### DISCUSSION TITLE
为什么我用docker镜像部署的apollo，portal页面只有配置导出，无配置导入

### DISCUSSION THREAD

**Discussion by lzx13283903316** [apolloconfig/apollo#5100] [General]
部署后的portal页面如下
![export](https://github.com/apolloconfig/apollo/assets/56867434/7ec6a717-8e6b-4ad7-8d24-1e075d4134b5)
![download](https://github.com/apolloconfig/apollo/assets/56867434/3b307c5c-eeb3-4cb4-9d5b-1dde17e93de9)
使用v2.1.0和v2.2.0版本的镜像都试了，都是同样的问题。
使用的数据库版本是mysql5.7.36，数据库初始化时分别使用的对应版本的sql文件。
有谁遇到和我一样的问题吗？是docker镜像的问题，还是数据库的问题，还是初始化数据库的脚本的问题？

**Comment by nobodyiam** [comment]
1. 缓存刷新下看看
2. 直接访问`/config_export.html`看看是否能进入

**Reply by lzx13283903316** [reply]
使用无痕进入还是只有导出，没有导入
![image](https://github.com/apolloconfig/apollo/assets/56867434/d704087a-2bd4-4fae-87e0-0c8aa67b9002)

**Reply by lzx13283903316** [reply]
找到问题了，服务器环境从新装了一遍，按照之前的流程从新操作了一遍，唯一的区别是没设置docker镜像加速。
下载下来的apollo镜像大概都300M左右(之前配置完镜像加速后，拉取的latest版本镜像大概都200M多点，拉取的2.1.0的都300M左右了，还以为优化了，镜像变小了，回想起来当时确实有问题)。
应该是设置的镜像加速地址问题！！！
![export](https://github.com/apolloconfig/apollo/assets/56867434/4edf978b-b8e0-4032-8e00-268174fc691f)
![download](https://github.com/apolloconfig/apollo/assets/56867434/62cc91ff-7c56-4e1d-9b69-42529296ce8c)
![images](https://github.com/apolloconfig/apollo/assets/56867434/01b4152b-9fe5-4078-9e75-b254df3fbab5)
